src/platform/imx8/include/platform/timer.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 NXP
+ *
+ * Author: Daniel Baluta <daniel.baluta@nxp.com>
+ */
+
+#ifndef __PLATFORM_TIMER_H__
+#define __PLATFORM_TIMER_H__
+
+#include <stdint.h>
+#include <sof/timer.h>
+#include <platform/interrupt.h>
+
+#define TIMER_COUNT	2
+
+/* timer numbers must use associated IRQ number */
+#define TIMER0		IRQ_NUM_TIMER0
+#define TIMER1		IRQ_NUM_TIMER1
+/* FIXME: we only support 2 timers */
+#define TIMER2		-1
```

Comment:
does SOF require 3 timers? What's the assumption here?

Reply:
> does SOF require 3 timers? What's the assumption here?

At some point yes. I see now that we can use less than three timers after commit 36d834855e0 (" xtensa: timer: Add support for less than three xtensa timers").

Will fix in v2.